internal/version/version.go

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2014-2022 Aerospike, Inc.
+ *
+ * Portions may be licensed to Aerospike, Inc. under one or more contributor
+ * license agreements.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License") you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package internal
+
+import (
+	"regexp"
+	"strconv"
+)
+
+type VersionStatus string
```

Comment:
Since the package name is version, the `Version` in the name is now redundant and unidiomatic.